k8s/prometheus-operator/values.yaml

```diff
@@ -122,31 +113,38 @@ kubeEtcd:
     insecureSkipVerify: true
     https: false
 kubeStateMetrics:
-  enabled: false
+  enabled: true
+kube-state-metrics:
+
 prometheus:
   prometheusSpec:
     serviceMonitorSelectorNilUsesHelmValues: false
     affinity:
       podAntiAffinity:
         requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchExpressions:
-            - key: app
-              operator: In
-              values:
-              - redis
-          topologyKey: "kubernetes.io/hostname"
+          - topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                    - redis
       podAffinity:
         requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchExpressions:
-            - key: app
-              operator: In
-              values:
-              - prometheus-operator-operator
-          topologyKey: "kubernetes.io/hostname"
+          - topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                    - prometheus-operator-operator
     nodeSelector:
       testground.nodetype: infra
+    disableCompaction: true
```

Comment:
The reason for this is that the block time is set to 2hours.

Reply:
I don't understand that. Why disable compaction? I understand that prometheus is holding the head block in memory for 2 hours, hence we should probably reduce the 2h to something like 15min, but I don't understand why disable the compaction?